### PR TITLE
Aasimar and Tiefling Description Update + Stat Rebalance

### PR DESCRIPTION
**Pull request by Xaytan**
## About The Pull Request
Updates Tiefling and Aasimar descriptions to match the current lore, and slightly adjusts species stats. Lore and stat changes requested by Centurion Salty. 

Descriptions and stats provided by Centurion Salty.

image

## Why It's Good For The Game

Updates to lore to reflect most current iteration.

Stat Changes:

Tieflings are now better suited to more roles. They will on average have 12.5% more limb health, allowing them to avoid being crippled longer and be crit less often. Returning their increased endurance makes them better fighters and labourers. The return of their -1 fortune restores a fairly important piece of flavor: The gods hate them for existing. Intelligence reduced by -1 to maintain stat total.

Aasimar:
Increased fortune to +2, reduced endurance to +0. This is to better differentiate them from their infernal cousins. The gods love them extra much now, and they'll be more likely to notice the fruits of their divine favor. It is very unlikely an aasimar will ever have a moment of bad luck no matter how poorly their day is going.

Test Proof
image
image

### CODE REVIEW

**Comment by Xaytan** [comment]
> I think we should not touch the fortune's race stat
> 
> There was a reason why tieflings received 0 fortune
> 
> image
> 
> Why +2 fortune for aasimars? Because of gods. Not an argument, sorry. Aasimars will be new rogue meta for better crits or something like this.
> 
> > Returning their increased endurance makes them better fighters and labourers. The return of their -1 fortune restores a fairly important piece of flavor
> 
> Tieflings will be even worse in combat. Fortune stats really affects on combat greatly. They're already shunned, do not touch their fortune stats or make it zero for all.
> 
> > Tieflings, while nominally mortal, are incapable of reproduction outside of their race.
> 
> Actually they can reproduce outside of their race by code. You can get pregnant by a dwarf or a drow as a tiefling.
> 
> Compromise/best solution: make fortune stat -1 for tiefling, +1 for aasimars as it was
> 
> **Think about not only lore but gameplay too.**

Wow was mistaken and did not bother to check the actual effects of the stat, instead relying on the claims of players on discord. Negative fortune has no effect on your probability of receiving critical hits.


Negative fortune's ONLY effects are:

Increased chance of failure when repairing an item with no skills.
Worse lockpicking probabilities.
5% chance per missing point of luck to hurt yourself when disarming a beartrap
Increased probability of being spotted when sneaking past NPCs
Increased chance of hurting yourself when using a dummy.
4% chance per missing luck to miss any attack. This is the only one that matters for combat.

And of course loss of things that benefit from having positive luck.

As for the critical bonus: 2% chance per positive fortune to get a +10 effective damage for applying wounds and crits. This is rolled twice, once for the wound and once for the crit, and the odds of rolling both even with a full 20 luck is only 4%. 

There is no universe in which +2 fortune makes aasimar meta for anything, considering their -2 con and -1 str. Even mining, which benefits immensely from fortune, they will be outshined by horcs and dwarves due to the strength penalty causing them to mine so much slower as to make their fortune insignificant.

**Comment by Vladmar11** [comment]
Is it worth reducing the luck of the Tieflings considering they are a shunned race due to the "excellent" lore? They have very limited roles as it is. They should have at least some advantage, at least in good stats.

File: code/modules/mob/living/carbon/human/species_types/roguetown/other/aasimar.dm [hunk 1]
```diff
@@ -5,14 +5,14 @@
...
-	They bear the mark of their celestial touch through many varying physical features. \
-	Their looks resemble the traditional characteristics of whichever of the Gods the Angel parent was associated with. \
-	Most commonly, Aasimar are similar to Humans, albeit taller, and possess uncanny beauty. \
-	They have strangely colored skin and are more physically frail than the average Human. \
-	Because of their upbringing, they make for natural conduits for godly powers. \
-	Rockhills populace holds them with a mixture of uneasy fear or, and respect. \
-	It is also widely believed that an Aasimars death is a bad omen..."
+	Aasimar are born of a rare union between mortals and Celestials of the Ten. \
+    They bear the mark of their celestial touch through many varying physical features. \
+    Their looks resemble the traditional characteristics of whichever of the Gods the Celestial parent was associated with. \
```
Comment (Gyrandola) [inline code/modules/mob/living/carbon/human/species_types/roguetown/other/aasimar.dm:10]: Celestials from the Marvel Cinematic Universe?
Was this even approved by a lore maintainer?

**Comment by wow50000** [comment]
> > I think we should not touch the fortune's race stat
> > There was a reason why tieflings received 0 fortune
> > image
> > Why +2 fortune for aasimars? Because of gods. Not an argument, sorry. Aasimars will be new rogue meta for better crits or something like this.
> > > Returning their increased endurance makes them better fighters and labourers. The return of their -1 fortune restores a fairly important piece of flavor
> > 
> > 
> > Tieflings will be even worse in combat. Fortune stats really affects on combat greatly. They're already shunned, do not touch their fortune stats or make it zero for all.
> > > Tieflings, while nominally mortal, are incapable of reproduction outside of their race.
> > 
> > 
> > Actually they can reproduce outside of their race by code. You can get pregnant by a dwarf or a drow as a tiefling.
> > Compromise/best solution: make fortune stat -1 for tiefling, +1 for aasimars as it was
> > **Think about not only lore but gameplay too.**
> 
> Wow was mistaken and did not bother to check the actual effects of the stat, instead relying on the claims of players on discord. Negative fortune has no effect on your probability of receiving critical hits and does not make you any more fragile.
> 
> Negative fortune's ONLY effects are:
> 
> Increased chance of failure when repairing an item with no skills. Worse lockpicking probabilities. 5% chance per missing point of luck to hurt yourself when disarming a beartrap Increased probability of being spotted when sneaking past NPCs Increased chance of hurting yourself when using a dummy. 4% chance per missing luck to miss any attack. This is the only one that matters for combat.
> 
> And of course loss of things that benefit from having positive luck.
> 
> As for the critical bonus: 2% chance per positive fortune to get a +10 effective damage for applying wounds and crits. This is rolled twice, once for the wound and once for the crit, and the odds of rolling both even with a full 20 luck is only 4%.
> 
> There is no universe in which +2 fortune makes aasimar meta for anything, considering their -2 con and -1 str. Even mining, which benefits immensely from fortune, they will be outshined by horcs and dwarves due to the strength penalty causing them to mine so much slower as to make their fortune insignificant.

honestly not too arsed about either statblock change, both seem decent but idk why aasimar get +2 luck

File: code/modules/mob/living/carbon/human/species_types/roguetown/other/aasimar.dm (hunk 1 above)
Comment (Xaytan) [reply code/modules/mob/living/carbon/human/species_types/roguetown/other/aasimar.dm:10]: Ask Centurion, he provided the description.

**Comment by Brutishcrab51** [comment]
Hello!

The stats and write-up presented have been reviewed by Lord Preston and MaiaOfMorgoth.

Celestial refers to the strictly DnD-esque/Pathfinder term for the wide array of 'angelic' entities that exist in the world of Grimmoria. We're stepping away from Abrahamic terms. Not strictly 'angels'.

Additionally, the old write-up for Tieflings said they couldn't reproduce, we just didn't change that bit. If it can be blocked via code, good. It should be.

-Centurion

**Comment by Brutishcrab51** [comment]
> Is it worth reducing the luck of the Tieflings considering they are a shunned race due to the "excellent" lore? They have very limited roles as it is. They should have at least some advantage, at least in good stats.

This is actually, as I see it, a buff for Tieflings. They will gain a negative Fortune modifier, but in the same update, they will gain Endurance, and their Constitution debuff will be reduced.

**Review by Gyrandola — APPROVED** [review]

**Comment by n0room** [comment]
While this is open, I have to say that emphasis on luck is not really any good thing for anyone. I've noticed a problem on the server that roll checks act in intervals. That is, if you roll, you'll get a successful tick in which every action you take is successful, and exactly the opposite with a failed tick. You could have a whole line of critical failures while crafting a seemingly easy item for your character, try to hit someone and critically miss more than once, or just successfully pick a lockpick without any failures for a good amount of time with a weak skill. Most likely the machine uses the same roll for an action after the previous one, not having time to generate a new roll.

I basically find rolls to be a weakness of the server, sometimes working so absurdly that you can never be sure about them, so I think increased or decreased luck is a major buff or debuff by no means. Tieflings are many times better and more playable than aasimars right now, hence the conclusion that they are played far less.

**Pull request merged**